__init__.py

```diff
@@ -183,17 +185,21 @@ def getDBPath():
     )
     if not db_path:
         raise IOError
-    db_path = unicode(db_path)
+    db_path = str(db_path)
     return db_path
 
 
 def getKindleVocabPath():
     try:
-        if platform == "win32":
+        if platform.system == "Windows":
             for l in string.ascii_uppercase:
                 path = r"{}:\\system\vocabulary\vocab.db".format(l)
                 if os.path.exists(path):
                     return r"{}:\\system\vocabulary".format(l)
+        elif platform.system == "Darwin":
+            path = "/Volumes/Kindle/system/vocabulary/vocab.db"
+            if os.path.exists(path):
+                return path
```

Comment:
This function should return directory path, not db file path

Reply:
interesting - I'll check this out as well. I was just attempting to follow the pattern of the other ones but didn't have my kindle mounted at the time so it wasn't working (making for a bad test). I'll fix it up and make sure this bit actually works